README.md

```diff
@@ -1,12 +1,13 @@
 # MongoDB Atlas Operator
-![MongoDB Atlas Operator](https://github.com/mongodb/mongodb-atlas-kubernetes/workflows/Test/badge.svg)
+[![MongoDB Atlas Operator](https://github.com/mongodb/mongodb-atlas-kubernetes/workflows/Test/badge.svg)](https://github.com/mongodb/mongodb-atlas-kubernetes/actions/workflows/test.yml?query=branch%3Amain)
 [![MongoDB Atlas Go Client](https://img.shields.io/badge/Powered%20by%20-go--client--mongodb--atlas-%2313AA52)](https://github.com/mongodb/go-client-mongodb-atlas)
 
 Welcome to the MongoDB Atlas Operator - a Kubernetes Operator which manages MongoDB Atlas Clusters from Kubernetes.
 
-> Current Status: *alpha*. We are working on an initial set of features to give users the ability to provision
```

Comment:
@theburi what's your opinion on this - we can write `*trial (beta)*` or just `*trial*`?

Reply:
MongoDB Atlas Operator (trial version)